Program.cs

```diff
@@ -1,7 +1,21 @@
 ﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+   internal class Program {
+      static void Main (string[] args) {
+         Random randomNum = new ();
+         int[] numArr = new int[6];
+         for (int i = 0; i < 6; ++i)

```

Comment:
Why limiting the app only to array length of 6? It could be simply enhanced by using a variable like 'arrCapacity' and referencing it appropriately in other places.

You could even get the input from user for arrCapacity?

Reply:
Get the user input for arrCapacity.